tests/Bridge/Doctrine/Orm/Filter/DateFilterTest.php

```diff
@@ -124,14 +126,14 @@ public function provideApplyTestData(): array
             ],
             'after' => [
                 [
-                    'dummyDate' => true,
+                    'dummyDate' => null,
```

Comment:
Because this is `$nullManagement`, which can only be `string` or `null`, not `bool`.